core/kogitoservice/deployer_resources.go

```diff
@@ -70,6 +70,8 @@ func (s *serviceDeployer) createRequiredResources() (resources map[reflect.Type]
 		return resources, err
 	}
 
+	// TODO: refactor this entire file
```

Comment:
can we have a jira opened for this? :)

Reply:
Yes! I completely forgot. https://issues.redhat.com/browse/KOGITO-4801